lib/iex/lib/iex/helpers.ex

```diff
@@ -533,14 +533,37 @@ defmodule IEx.Helpers do
 
   defp print_runtime_info_topic(:memory) do
     print_pane("Memory")
-    print_memory("Total", :total, :MB)
+    print_memory("Total", :total)
     print_memory("Atoms", :atom)
     print_memory("Binaries", :binary)
     print_memory("Code", :code)
     print_memory("ETS", :ets)
     print_memory("Processes", :processes)
   end
 
+  defp print_runtime_info_topic(:memory_details) do
+    print_pane("Memory Details")
+    print_memory_detail("Atom Space", :atom_space)
+    print_memory_detail("Atom Table", :atom_table)
+    print_memory_detail("BIF Timer", :bif_timer)
+    print_memory_detail("Bits Bufs Size", :bits_bufs_size)
+    print_memory_detail("Dist Table", :dist_table)
+    print_memory_detail("ETS Misc", :ets_misc)
+    print_memory_detail("Export List", :export_list)
+    print_memory_detail("Export Table", :export_table)
+    print_memory_detail("Function Table", :fun_table)
+    print_memory_detail("Link LH", :link_lh)
```

Comment:
I could not find any info on what that might be. Could use a friendlier name?

Reply:
Ping @michalmuskala.

Reply:
FWIW it's `N/A` on Erlang/OTP 21.0

Reply:
That's probably related to the links being largely reworked to increase scalability.

Reply:
@aerosol even better, let's remove it then.